landing-pages/site/content/en/announcements/_index.md

```diff
@@ -1355,7 +1355,7 @@ Apache Airflow graduates from the Incubator and is now a TLP!
     - [ASF "Foundation" Blog](https://s.apache.org/LeeE)
     - [The ASF Twitter Feed](https://twitter.com/TheASF/status/1082593312421208064)
     - [The ASF on LinkedIn](https://www.linkedin.com/company/the-apache-software-foundation)
-    - You can find more announcements on our twitter account as well : [@ApacheAirflow](https://twitter.com/ApacheAirflow)
+    - You can find more announcements on our Twitter account as well : [@ApacheAirflow](https://twitter.com/ApacheAirflow)
```

Comment:
We can remove this line - we deleted our Twiitter/X account. Actually it would be great to remove all the references to Twitter / X since we ar touching it.

Reply:
I did it now in this file and made a global search on the repo: no other mention found.